### PR TITLE
Working sound-obj component, with test audio

### PR DESCRIPTION
**Pull request by azimpradhan**

### CODE REVIEW

**Comment by azimpradhan** [comment]
@felakuti4life, I tried to cherry-pick your commits and squashed them down into one commit. When running the branch and going to the url localhost:3000/tester, I did not see or hear anything. The same applies when I checked out your ethan branch. What does one need to do to see the sound-obj component in action?

**Comment by azimpradhan** [comment]
@mattahorton and @felakuti4life 
Here are some tips on how we can efficiently create clean commits and focus the majority of our time on development:
After creating a PR, delete your old branch. This is because, we want to create commits on top of what stage already has. To do this you want to checkout the latest copy of stage.

to delete your old remote branch do:

```
git push origin --delete <-BranchName->
```

to delete your old local branch do:

```
git branch -D <-BranchName->
```

now check out a new branch off the latest stage:

```
git fetch origin
git branch -b <-your_github_handle->/<-new_feature_name-> origin/stage
```

Go ahead and push this branch to make it exist remotely:

```
git push origin <-your_github_handle->/<-new_feature_name->
```

Now you are ready to develop. When developing try to create clean commits. For example, suppose I am creating database migrations and also modifying the title in an html page. when I type:

```
git status
```

the files that I have changed are:

```
M db/migrate/234232.rb
M db/migrate/2383832.rb
M frontend/index.html
```

If I am ready to commit these files, I will create separate commits by adding and comitting the db/migrate files separate from the index.html file.

After you have committed your files, it is important that you fetch the latest copy of the stage branch and rebase with stage. By rebasing with stage, you are bringing in all commits that have been merged into stage (good commits) and laying your commits on top of them:
for example, 
let's say when you start development stage has commits: 

```
A, B, C
```

you create commits X, Y, so your branch looks like this: 

```
A, B, C, X, Y
```

However another teammate merged commits D and E into stage, so stage now look like: 

```
A, B, C, D, E
```

By rebasing with stage, you will make your branch go from looking like 

```
A, B, C, X, Y 
```

to looking like 

```
A, B, C, D, E, X, Y
```

The idea is that your commits will appear as the last set of commits.

to get the latest copy of stage, do 

```
git fetch origin
```

this updates updates the copies of all remote branches locally

now rebase your branch with origin/stage

```
git rebase -i origin/stage
```

Finally, you'll want to update your remote branch to include the changes after the rebase.

```
git push origin <-your_github_handle->/<-new_feature_name->
```

If no one had updated stage before your rebase, the above commit should have worked fine. However, if stage was changed, then github will not be happy with your push. Nothing to worry about. If the rebase with stage was successful you definitely want the commit history of your current local branch to be the one that is used. Therefore you can:

```
git push origin <-your_github_handle->/<-new_feature_name-> --force
```

This forces the contents of your local branch onto the remote branch. To refer to the example above your remote branch should now contain commits:

```
A, B, C, D, E, X, Y 
```

while stage contains commits:

```
A, B, C, D, E
```

Now you can create your PR. Once your PR is reviewed by yourself and anyone you ask to review it, pressing the merge button, will append your commits (X, and Y), to the end of stage's commit history

note: DO NOT merge another person's branch into your branch. This makes it almost impossible to create a PR that can be merged. (each person should focus on being able to merge their own PR with stage)

**Comment by felakuti4life** [comment]
@azimpradhan , as for the tester, I don't commit that to the repo, since that was just a blank template I use to test the component.

To get a sound-obj up and running, create one in any template:
`{{sound-obj id='newWave' fname='http://localhost:3000/assets/telephone.wav'}}`
`id` is the actual id for the div the component generates, and fname is the url to the audio file you're loading. If you're updating off of my branch, there should be a sample audio file that gets copied onto the dist on build.

**Comment by felakuti4life** [comment]
@azimpradhan 
Ok to to merge?

File: frontend/app/router.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('tester');
```
Comment (felakuti4life) [inline frontend/app/router.js]: this can be deleted in stage

**Pull request merged**